warehouse/migrations/versions/bb986a64761a_rename_githubprovider_fields.py

```diff
@@ -0,0 +1,67 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+rename GitHubProvider fields
+
+Revision ID: bb986a64761a
+Revises: 9f0f99509d92
+Create Date: 2022-04-22 22:00:53.832695
+"""
+
+import sqlalchemy as sa
+
+from alembic import op
+
+revision = "bb986a64761a"
+down_revision = "9f0f99509d92"
+
+
+def upgrade():
+    op.add_column(
+        "github_oidc_providers",
+        sa.Column("repository_owner", sa.String(), nullable=True),
+    )
+    op.add_column(
+        "github_oidc_providers",
+        sa.Column("repository_owner_id", sa.String(), nullable=True),
+    )
+    op.drop_constraint(
+        "_github_oidc_provider_uc", "github_oidc_providers", type_="unique"
+    )
+    op.create_unique_constraint(
+        "_github_oidc_provider_uc",
+        "github_oidc_providers",
+        ["repository_name", "repository_owner", "workflow_filename"],
+    )
+    op.drop_column("github_oidc_providers", "owner_id")
+    op.drop_column("github_oidc_providers", "owner")
```

Comment:
Doesn't actually matter for the production DB right now because there are no rows in this table, but probably better to make this a rename instead. Generally we don't want to drop/add columns.

Reply:
Good to know, will tweak.

Reply:
(I wish alembic would catch these renames, but thinking about it more that wouldn't be possible in the general case...)

Reply:
[dddb25d](https://github.com/pypa/warehouse/pull/11239/commits/dddb25df9df8ba89318f165148f6e727b75da1f9)